ui/components/component-library/button-link/README.mdx

```diff
@@ -15,10 +15,6 @@ The `ButtonLink` is an extension of `ButtonBase` to support link styles
 
 <ArgsTable of={ButtonLink} />
 
-The `ButtonLink` accepts all [ButtonBase](/docs/components-componentlibrary-buttonbase--default-story#props) component props
-
-<ArgsTable of={ButtonBase} />
-
 ### Size
```

Comment:
Removing as it repeated the props in the prop table that wasn't needed

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/c93d4ff0-1fe0-4c64-9920-7ed1518e385c



### After


https://github.com/MetaMask/metamask-extension/assets/8112138/7c15b605-f5be-444f-9a67-fe72383dec9e